src/applications/ivc-champva/10-7959a/config/form.js

```diff
@@ -59,6 +59,18 @@ const formConfig = {
       saved: 'Your CHAMPVA claim form application has been saved.',
     },
   },
+  preSubmitInfo: {
+    statementOfTruth: {
+      body:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      messageAriaDescribedby:
+        'I confirm that the identifying information in this form is accurate and has been represented correctly.',
+      fullNamePath: formData =>
+        formData?.certifierRole === 'applicant'
+          ? 'applicantName'
+          : 'certifierName',
```

Comment:
Depending on role, require that the signature match who's actually filling out the form